### PR TITLE
Null Transform Plugin Autest finished

### PR DESCRIPTION
**Pull request by dyrock**
Autest files for null transform plugin

### CODE REVIEW

**Comment by persiaAziz** [comment]
[approve ci]

File: tests/gold_tests/null_transform/null_transform.test.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+# Add response the server dictionary
+server.addResponse("sessionfile.log", request_header, response_header)
+ts.Disk.records_config.update({
+    'proxy.config.diags.debug.enabled': 1,
+    'proxy.config.diags.debug.tags': 'null-transform'
+})
+
+# Load plugin
+ts.Disk.plugin_config.AddLine(
+    'null-transform.so'
```
Comment (persiaAziz) [inline tests/gold_tests/null_transform/null_transform.test.py]: you will need to provide the absolute path here. Ci can not find the plugin and it is complaining.

Comment (dyrock) [inline tests/gold_tests/null_transform/null_transform.test.py]: I was about to ask how to locate the absolute path for .so file. By default they are installed to libexec/trafficserver? And it is one of the example plugin.

Comment (persiaAziz) [inline tests/gold_tests/null_transform/null_transform.test.py]: yes by default these are installed in libexec/trafficserver. But looks like the mechanism to copy example plugins is either missing or not working properly in AuTest. Do you see this plugin in your ts folder under sandbox?

Comment (dyrock) [inline tests/gold_tests/null_transform/null_transform.test.py]: Yes, I can see it in ts/libexec/trafficserver.

**Comment by persiaAziz** [comment]
[approve ci autest]

**Comment by shinrich** [comment]
You'll want to make your first PR against master instead of 7.1.x.

**Pull request closed without merging**